DataFormats/Math/interface/deltaPhi.h

```diff
@@ -38,7 +44,13 @@ namespace reco {
 
   template <typename T>
   constexpr T deltaPhi(T phi1, T phi2) {
-    return reduceRange(phi1 - phi2);
+    std::cout << std::fixed << std::setprecision(20) << "phi1=" << phi1 << "\nphi2=" << phi2 << '\n';
```

Comment:
I suppose this should be removed

Reply:
It will be once I determine what is still causing the segfault when running with cmsRunVDT.

Reply:
May I suggest not to push to the remote until the cause is understood. (or close this PR in the interim)